cms/djangoapps/contentstore/views/program.py

```diff
@@ -27,7 +28,8 @@ def get(self, request, *args, **kwargs):
             return render_to_response('program_authoring.html', {
                 'show_programs_header': programs_config.is_studio_tab_enabled,
                 'authoring_app_config': programs_config.authoring_app_config,
-                'programs_api_url': programs_config.public_api_url,
+                'lms_base_url': '//{}/'.format(settings.LMS_BASE),
+                'programs_api_url': programs_config.public_service_url,
```

Comment:
It will so I shall create that PR too and make the change here (so hopefully we can make Monday's RC)